webui/react/src/e2e/tests/navigation.spec.ts

```diff
@@ -59,11 +60,11 @@ test.describe('Navigation', () => {
     });
 
     await test.step('Navigate to Workspaces', async () => {
-      await page.getByRole('link', { name: 'Workspaces' }).click();
-      const expectedURL = /workspaces/;
-      await page.waitForURL(expectedURL);
-      await expect.soft(page).toHaveTitle('Workspaces - Determined');
-      await expect.soft(page).toHaveURL(expectedURL);
+      const workspacesPage = new Workspaces(page);
+      await workspacesPage.nav.sidebar.workspaces.pwLocator.click()
+      await page.waitForURL(workspacesPage.url);
+      await expect.soft(page).toHaveTitle(workspacesPage.title);
+      await expect.soft(page).toHaveURL(workspacesPage.url);
```

Comment:
not sure how this is different from line 65

Reply:
removed